lib/iris/tests/unit/analysis/trajectory/test__nearest_neighbour_indices_ndcoords.py

```diff
@@ -141,13 +140,9 @@ def test_sample_dictionary(self):
         cube.add_aux_coord(co_y, 0)
         cube.add_aux_coord(co_x, 1)
         sample_point = {'x': 2.8, 'y': 18.5}
-        warn_call = self.patch(
-            'iris.analysis._interpolate_private.warn_deprecated')
-        result = nn_ndinds(cube, sample_point)
-        self.assertEqual(result, [(1, 2)])
-        self.assertEqual(warn_call.call_count, 1)
-        self.assertIn('dictionary to specify points is deprecated',
-                      warn_call.call_args[0][0])
+        exp_emsg = 'must be a list of \(coordinate, value\) pairs'
+        with self.assertRaisesRegexp(TypeError, exp_emsg):
+            nn_ndinds(cube, sample_point)
```

Comment:
Is this the only test we have for nn_ind_ndcoords?

Reply:
Well, there's this whole test module that I've moved... This was the only bit that actively needed updating though, as I've removed a [little bit of deprecated functionality](https://github.com/SciTools/iris/pull/2804/files#diff-372eebf3389e425b54bf8e165cc46d2eL220).

Reply:
Yeah that makes sense.  Good.  I'm happy.